src/main/scala/LetterSquare.scala

```diff
@@ -0,0 +1,22 @@
+object LetterSquare extends App {
+  def square(size:Int, letter:String){
+    for(i <- 1 to size){
+      print(letter)
+      if(i == size){
+        println()
+      }
+    }
+  }
+
+  println("Enter number: ")
+  var number = scala.io.StdIn.readInt()
```

Comment:
you're not reassigning the value so it should be val, similar for the letter variable

Reply:
I changed the number variable to val but couldn't with the letter because I needed to transform it to uppercase.